app/models/rule_group.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# OpenSCAP RuleGroup
+class RuleGroup < ApplicationRecord
+  # Need to set primary key format to work with uuid primary key column
+  has_ancestry(primary_key_format: /\A[\w\-]+(\/[\w\-]+)*\z/)
+
+  belongs_to :benchmark, class_name: 'Xccdf::Benchmark'
+  has_many :rules
+  has_many :required_rules, through: :rule_group_rules, source: :rule
```

Comment:
Wait, this is a bit contradictive with what you stated in the comment above :thinking: rule groups have required/conflicting rule groups and rules have required/conflicting rules? Because the code is written the opposite way :thinking:

Reply:
Right, I only have half of it done(or pushed to this PR at least)
Rules can have required/conflicting rule groups or rules and RuleGroups can have required/conflicting rules or rule groups
I only have rule groups with required/conflicting rules and rules with required/conflicting rule groups set up in this PR for now but I have the other other way in progress on my branch so I'll push that after I finish it and test!
Does that make sense?

Reply:
Yup, thanks!